docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -1882,4 +1884,4 @@ items:
 - name: Level 5 warning messages
   items:
   - name: CS8892
-    href: cs8892.md
+    href: warning-waves.md#cs8892---method-will-not-be-used-as-an-entry-point-because-a-synchronous-entry-point-method-was-found
```

Comment:
This section should be removed and replaced with the warning waves

Line 1384 and 1385 should go here. In addition, the `displayName` metadata should list all the warning IDs in that file.

Should I open a separate PR for this?